db/migrate/20220314204321_add_number_of_available_kudos_to_employees.rb

```diff
@@ -0,0 +1,6 @@
+class AddNumberOfAvailableKudosToEmployees < ActiveRecord::Migration[6.1]
+  def change
+    add_column :employees, :number_of_available_kudos, :integer, null: false, default: '10'
+    add_index :employees, :number_of_available_kudos
```

Comment:
this is not needed - are you going to search for an employee by his number of available kudos? :)

Reply:
Corrected. Thanks for the hint ;)